zerver/models.py

```diff
@@ -412,10 +412,54 @@ class Realm(models.Model):
     # Messages older than this message ID in the organization are inaccessible.
     first_visible_message_id: int = models.IntegerField(default=0)
 
-    # Valid org_types are {CORPORATE, COMMUNITY}
-    CORPORATE = 1
-    COMMUNITY = 2
-    org_type: int = models.PositiveSmallIntegerField(default=CORPORATE)
+    # Valid org_types
+    ORG_TYPES: Dict[str, Dict[str, Any]] = {
+        "business": {
+            "name": "Business",
+            "id": 1,
+        },
+        "community": {
+            "name": "Community",
+            "id": 2,
+        },
+        "opensource": {
+            "name": "Open-source project",
+            "id": 3,
+        },
+        "research": {
+            "name": "Research",
+            "id": 4,
+        },
+        "event": {
+            "name": "Event or conference",
+            "id": 5,
+        },
+        "nonprofit": {
+            "name": "Non-profit",
+            "id": 6,
+        },
+        "personal": {
+            "name": "Personal",
+            "id": 7,
+        },
+        "other": {
+            "name": "Other",
+            "id": 8,
+        },
+        "political_group": {
+            "name": "Political group",
+            "id": 9,
+        },
+        "education": {
+            "name": "Education",
+            "id": 10,
+        },
+    }
+
+    OrgType = models.IntegerChoices("OrgType", list(ORG_TYPES.keys()))
+    org_type: int = models.PositiveSmallIntegerField(
+        default=ORG_TYPES["business"]["id"], choices=OrgType.choices
+    )
```

Comment:
How does this `IntegerChoices` thing work?  We seem to not have passed the `id` values into `OrgType` at all, so is it important to correctness that the ID values be sequential?  That deserves at least a comment, since it's very counterintuitive for a dictionary to work that way; we may also just want to do another pattern. 

@sahil839 in case you have ideas for how to do this more cleanly.

Reply:
I looked at djanog docs, `IntegerChoices` returns a list of tuples of form [(1, 'Business'), (2, 'Community')...], we just have passed a list of strings and then the integers are assigned to them in order as per https://docs.djangoproject.com/en/3.2/ref/models/fields/#enumeration-types. Only integers are used for validation and the string is for reference/naming purposes. And the correct thing to pass as choices is a list of tuples.

But I am not sure that we actually need this or not, as we do not do any validation on DB side for other settings which has list of integers as possible values, we just validate them when changing through API.  Also the validation thriough choices field will not take place on `.save` method, it will take place only something like `.full_clean` (as mentioned [here](https://docs.djangoproject.com/en/3.2/ref/models/instances/#validating-objects)) which we use only in rare cases.

Reply:
A small correction, OrgType.choices is a list of tuples.

Reply:
@sahil839 I think I may have found a cleaner way to do all this. Thanks for looking into this for me, I appreciate it! :)

Reply:
After trying everything with `IntegerChoices`, I just removed it and passed in a list of tuples for the `choices` parameter directly to the `PositiveSmallIntegerField`. Huge thanks to @sahil839 for looking into this for me!